GameData/RealismOverhaul/RO_SuggestedMods/NearFutureTechnologies/RO_NearFuture_Electrical_Nuclear.cfg

```diff
@@ -256,6 +256,164 @@
 	}
 }
 
+// TOPAZ-I reactor - main source: https://inis.iaea.org/collection/NCLCollectionStore/_Public/25/070/25070118.pdf
++PART[reactor-125]:FOR[RealismOverhaul]
+{
+	@name = RO-reactor-TOPAZI
+	%RSSROConfig = True
+	@title = TOPAZ-I Nuclear Reactor
+	@description = Thermionic reactor, used in the last 2 Soviet RORSAT satellites. Intended lifetime of 1 year.
+	@mass = 1.137	//Assumed about 1 ton heavier than TOPAZ-II
```

Comment:
Not sure if the mass figure is wrong or if the comment is wrong, but this is only 100kg heavier than TOPAZ-II